src/lib/moment/constructor.js

```diff
@@ -58,7 +58,7 @@ var updateInProgress = false;
 // Moment prototype object
 export function Moment(config) {
     copyConfig(this, config);
-    this._d = new Date(config._d.getTime());
+    this._d = new Date(config._d != null ? config._d.getTime() : NaN);
```

Comment:
No. `!= null` catches `null` and `undefined`. In our case its actually `undefined``.